frontend/package.json

```diff
@@ -24,6 +22,9 @@
     "testLib": "yarn workspace @streamlit/lib testWatch",
     "testApp": "yarn workspace @streamlit/app testWatch"
   },
+  "resolutions": {
+    "nwsapi": "2.2.9"
```

Comment:
it could make sense to add a comment here to briefly explain what we are pinning here and why

Reply:
Won't be able to add a comment cause it's JSON, but I can explain it's tied to this issue https://github.com/dperini/nwsapi/pull/125 affecting one of our tests. I was originally planning on modifying the test, but the issue hints at the next update being able to handle it better, so hopefully it's shortlived.